scripts/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+echo "Build the RWD Docker container."
+
+BUILD_COMMAND="docker build -t docker-rwd ."
+
+echo "$BUILD_COMMAND"
```

Comment:
For context, the bundle script that this was copied from purposely did not use `-x` since 99% of the bash output would be noise (only the final command was relevant).